test/libsolidity/syntaxTests/types/too_large_numbers.sol

```diff
@@ -0,0 +1,11 @@
+contract C {
+  uint8[-21111111111121 * 12222222222202 ** -22] a;
+  uint8[12222222222202 ** -22 * -21111111111121] b;
```

Comment:
Can you find a minimal example, please? I would assume something like `1e-100` would work.

Reply:
Oh and please rename the test so that it is clear that this is about array sizes.

Reply:
I'm working on getting it as minimal as possible, although, I *think* arrays are just the carrier, as I was able to reproduce the solc-hanging also with simple `fixed a = 12222222222202 ** -22 * -21111111111121;` too. But I'll digg into it again tomorrow.

Reply:
@chriseth the smallest test case I have found so far to trigger this hang is `var a = -1 / (1e10 ** 10);`